src/robot/model/control.py

```diff
@@ -117,7 +117,7 @@ def _include_in_repr(self, name, value):
         return name == 'condition' or value is not None
 
     def to_dict(self):
-        data = {'type': self.type}
+        data: Dict[str, Any] = {'type': self.type}
```

Comment:
This doesn't look too useful, especially inside a method. It would be better to add a type hint to the method, but I believe just `dict` is good enough unless we'd use a `TypedDict` to properly document the structure. Using a `TypedDict` with all `to_dict` methods would be a huge task and not in the scope of this PR. It would probably be better to use something like pydantic for documenting the `to_dict/json` and `from_dict/json` methods anyway.